css/bem-css.md

```diff
@@ -38,58 +40,58 @@ We do this:
 
 We _don't_ do this:
 ```css
-.feature
-.content
-.btn
+.c-feature
+.c-content
+.c-btn
 ```
 
 We do this:
 ```css
-.alert-box
-.button
+.c-alert-box
+.c-button
 ```
 
 **Element**: styles that only apply to children of a block. Elements can also be blocks themselves. Class name is a concatenation of the block name, two underscores and the element name. Examples:
```

Comment:
`Elements can also be blocks themselves`
Think this might be a bit misleading. We wouldn't accept someone treating a BEM element as a block and then creating an element on the back of that e.g. `c-alert-box__close__link`.